ajax/libs/material-datetime-picker/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "material-datetime-picker",
+  "filename": "material-datetime-picker.min.js",
+  "version": "2.4.0",
+  "description": "A Vanilla JS Material Design date/time picker component",
+  "homepage": "https://ripjar.github.io/material-datetime-picker/",
+  "keywords": [
+    "material-design",
+    "date-picker",
+    "datetime-picker",
+    "javascript"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ripjar/material-datetime-picker"
+  },
+  "npmName": "material-datetime-picker",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*picker!(.mjs)*"
```

Comment:
Let's use `**/!(*.mjs*)`.
Although, the current pattern can grab the correct files, the patten can be revised better to read.
As https://github.com/cdnjs/cdnjs/pull/12201#pullrequestreview-78821928 this comment said.
Many thanks.